### PR TITLE
claimed domain enforcement rule

### PR DESCRIPTION
**Pull request by Luci2015**

### CODE REVIEW

File: docs/en/index.md [hunk 1]
```diff
@@ -7,14 +7,22 @@ nav_order: 15
...
-<p>As a result, any applications directly accessing the User Management API which include logic <strong>dependent on the product name</strong> will need to be updated. If you have not included the product name in the code, then this will not impact your connection to the User Management API. If you use the User Sync Tool, you should see no impact.</p>
+<p>As a result, any application directly accessing the User Management API which include logic <strong>dependent on the product name</strong> will need to be updated. If you have not included the product name in the code, then this will not impact your connection to the User Management API. If you use the User Sync Tool, you should see no impact.</p>
 <p>Note: the term “CC” included in the name of a Product Profile <em>will not change</em>. If you rely on the name of the “product admin group” (e.g., <code>_admin_&lt;product name&gt;</code>) you may be impacted and have to update your scripts.</p>
 <p>As a best practice, it is recommended to avoid any logic that expects fixed product names.</p>
 </div>
+<hr class="api-ref-rule">
```
Comment (adobeDavidT) [inline docs/en/index.md]: Would people expect to see the latest news first?

File: docs/en/index.md [hunk 2]
```diff
@@ -7,14 +7,22 @@ nav_order: 15
...
+<p>Starting August 8th 2019, a change in a dependent service of the UMAPI is applied to align with the multiple domains per directory model and tightening the security of the <strong>create</strong> and <strong>update</strong> APIs.</p>
```
Comment (adobeDavidT) [inline docs/en/index.md]: Maybe just "a change is applied to align..."?

File: docs/en/index.md (hunk 1 above)
Comment (Luci2015) [reply docs/en/index.md]: You're right! I'll fix to have thhe latest news on top.

File: docs/en/index.md (hunk 2 above)
Comment (Luci2015) [reply docs/en/index.md]: Sure, this will get changed as well.

**Pull request merged**